Apps/WebAPI/sdLitica.WebAPI/Controllers/v1/TimeSeriesController.cs

```diff
@@ -37,12 +38,80 @@ public TimeSeriesController(ITimeSeriesService timeSeriesService)
         ///    400 - There were issues with passed data, i.e. required fields missing or length constraints violated
         /// </returns>
         [HttpPost]
+        [AllowAnonymous]
         public IActionResult AddTimeSeries()
         {
             var t = _timeSeriesService.AddRandomTimeSeries();
             return Ok(t.Result);
         }
 
+        /// <summary>
+        /// This REST API handler creates a new time-series metadata object for current user
+        /// </summary>
+        /// <param name="timeSeriesModel"></param>
+        /// <returns>
+        ///    200 - Time-series was successfully created. Response payload will contain object with assigned id
+        ///    400 - There were issues with passed data, i.e. required fields missing or length constraints violated
+        /// </returns>
+        [HttpPost]
+        [Route("temp")]
```

Comment:
only temporarily for this MR. needed to ask, if I can delete methods written by Rodrigo, which pathes conflict with new methods. I think we don't need them now, because now the different thing is assumed by "metadata". also, method for adding random time-series is no longer needed. @Rodrigo, what do u think?

Reply:
Sorry for delay. I was totally off these days.  @jwesley777  feel free to remove unnecessary methods. No problem.

Reply:
thanks for the reply. will remove these methods in next updates